unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarShape.cs

```diff
@@ -142,14 +142,15 @@ private void OnWorldReposition(Vector3 current, Vector3 previous)
             MinimapMetadataController.i?.UpdateMinimapUserInformation(avatarUserInfo);
         }
 
-        private void OnEntityTransformChanged(DCLTransform.Model updatedModel)
+        private void OnEntityTransformChanged(object newModel)
         {
-            lastAvatarPosition = updatedModel.position;
+            DCLTransform.Model newTransformModel = (DCLTransform.Model)newModel;
```

Comment:
We should ensure the type is correct here as in my previous requestes.

Reply:
The only dispatcher of this event (`DCLTransform`) will always come with `DCLTransform.Model`. 

I took the shortcut of changing the event signature to pass `object` type because keeping the reference to the specific component from the entity will give problems in the future. This problems will materialize when trying to put `DCLTransform` in a single assembly for example.

This exposes a design flaw: we shouldn't be having component-specific events at the entity layer, and those events should belong elsewhere.

Reply:
Adding some tech debt to address this in the future.